### ISSUE TITLE
English instructions

### ISSUE THREAD

**Issue by frh3ddy** [EvilCult/moviecatcher#26]
can you provide English guide,

**Comment by EvilCult** [comment]
Hey, I so sorry for replying you so late.
The purpose of this project is download or watches videos on the cloud.
For the people who behind the "wall", and can't visit "google" and "netflix" and so on, but still wanna watch movies / tv shows. E.g. Game of Thrones

But, for several reasons, this project is dead (or dying, probably).
1. This project is basing a service called "BaiduYun", and recently, this service is hard to be used.(Unable to upload video due to copyright, and download speed is very slow)
2. Many websites(resource) have been offline.

So, basically, if you are speaking English, you probably don't need this.
